lib/db/redis.js

```diff
@@ -0,0 +1,75 @@
+var redis = require('redis')
+var jws = require('jws')
+var P = require('../promise')
+
+function match(filter, str) {
+  try { // too lazy
+    var payload = JSON.parse(jws.decode(str).payload)
```

Comment:
Not a concern right now, but I do worry about the overhead of decoding these JWTs every time we want to manipulate them.  We may want to extract out the filterable fields and store them "in the clear" somehow.